project/AppDependencies.scala

```diff
@@ -1,35 +1,33 @@
-import play.core.PlayVersion
-import play.core.PlayVersion.current
+import play.core.PlayVersion._
 import sbt._
 
 object AppDependencies {
 
-  private val mongoVersion = "0.71.0"
+  private val mongoVersion = "0.73.0"
+  private val bootstrapVersion = "7.8.0"
 
   val compile: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc"         %% "bootstrap-backend-play-28"       % "7.3.0",
+    "uk.gov.hmrc"         %% "bootstrap-backend-play-28"       % bootstrapVersion,
     "uk.gov.hmrc.mongo"   %% "hmrc-mongo-play-28"              % mongoVersion,
-    "com.typesafe.play"   %% "play-iteratees"                  % "2.6.1",
```

Comment:
We 100% sure we do not need these? :-)

Reply:
@ajchilts You're right to question it after the manage-documents mishap! This library has been archived, isn't cross-compatible for Scala 2.13 and seemingly isn't being used (the application compiles, tests pass etc.)

We had the same thing in the ETL service. The dependency has been removed and the scheduled job looks to be running successfully every hour - https://kibana.tools.staging.tax.service.gov.uk/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-1d,to:now))&_a=(columns:!(message),filters:!(),index:match_all_logstash_ingested_logs_kibana_index_pattern,interval:auto,query:(language:lucene,query:'app.raw:%22transit-movements-trader-reference-data-etl%22'),sort:!(!('@timestamp',desc)))

These dependencies were added in by Amaal back in August 2020 (https://github.com/hmrc/customs-reference-data/commit/f5dbda21038ebec5661bd02c5143e0a4766634b3). It's possible that they were never needed looking at that commit.